content/applications/finance/documents.rst

```diff
@@ -44,33 +44,54 @@ Workspaces
 
 Workspaces are hierarchical folders having their own set of tags and actions. Default workspaces
 exist, but you can create your own by going to :menuselection:`Documents --> Configuration -->
-Workspaces` and clicking on :guilabel:`Create`.
+Workspaces` and clicking on :guilabel:`New`. A new page pops up:
 
-.. note::
-   :guilabel:`Workspaces` and :guilabel:`Sub-workspaces` can be created, edited, or deleted by
-   clicking on the gear icon :guilabel:`⚙` on the left menu.
-
-.. image:: documents/sub-workspaces-creation.png
-   :align: center
-   :alt: Create sub-workspaces from the left menu
+- :guilabel:`Name`: add the name of your workspace;
+- :guilabel:`Parent Workspace`: if the workspace belongs to a parent workspace (and is then a
+  sub-workspace), select it from the list;
+- :guilabel:`Company`: if your workspace is only available to the selected company;
 
 Tags
-====
+----
 
 Tags are used within workspaces to add a level of differentiation between documents. They are
 organized per category and filters can be used to sort them.
 
+Tags can be created from the :guilabel:`Tags` tab. Click :guilabel:`Add a line`, create the
+:guilabel:`Tag Category`, and :guilabel:`Name` your tags. Go to the :guilabel:`Tooltip` tab to add
+informative text for when users hover over your tag.
+
 .. note::
    - The tags of a parent workspace apply to the child workspaces automatically.
    - Tags can be created and modified by going to :menuselection:`Configuration --> Tags`.
    - Tags can also be created, edited, or deleted, by clicking on the gear icon :guilabel:`⚙`, on
      the left menu.
 
+Access rights
+-------------
+
+From the :guilabel:`Access Rights` tab, you can add :guilabel:`Write Groups` that can see the
+workspace and read, create, and edit its documents. You can also add :guilabel:`Read Groups` to see
+the workspace and read its documents without creation and editing rights.
```

Comment:
specifiy more about where to find these settings, say to open the cog icon and access rights can be found in the edit section